test_elasticsearch_dsl/test_integration/test_document.py

```diff
@@ -60,6 +60,39 @@ def test_get_with_tz_date(data_client):
     tzinfo = timezone('Europe/Prague')
     assert tzinfo.localize(datetime(2014, 5, 2, 13, 47, 19)) == first_commit.authored_date
 
+def test_mget(data_client):
+    commit_ids = [
+        '0',
+        '3ca6e1e73a071a705b4babd2f581c91a2a3e5037',
+        'eb3e543323f189fd7b698e66295427204fff5755',
+    ]
+    commits = Commit.mget(commit_ids)
+    assert commits[0] is None
+    assert commits[1].id == '3ca6e1e73a071a705b4babd2f581c91a2a3e5037'
+    assert commits[2].id == 'eb3e543323f189fd7b698e66295427204fff5755'
+
+def test_mget_missing_docs(data_client):
+    commit_ids = [
+        '0',
+        '3ca6e1e73a071a705b4babd2f581c91a2a3e5037',
+        'eb3e543323f189fd7b698e66295427204fff5755',
+    ]
+
+    with raises(ValueError):
+        Commit.mget(commit_ids, missing='raj')
```

Comment:
can we split all of these into individual tests? that way, when one fails, we know where the error was instead of just knowing that something is wrong. I think this should be 4 tests instead of 2 (test with missing='none' is already implemented in `test_mget`, no need to duplicate it)